### ISSUE TITLE
Sigmoid output

### ISSUE THREAD

**Issue by Tarikyesiltepe** [bethgelab/foolbox#664]
When I apply adversarial attack to having one neuron output network, it does not supported. How can we solve it? Output neuron must be multiple and use softmax?

**Comment by zimmerrol** [comment]
Yes, that should be correct. I think the easist way to use the existing attacks for your model would be to add a second "virtual" output `y2` to your network (just for the attack) which is the negated version of your first output, i.e. `y2 = -y1`. Then you don't have to change anything in your network.

**Comment by jangop** [comment]
Does the single sigmoidal output neuron imply regression? Or is this about binary classification?
